package.json

```diff
@@ -51,13 +56,21 @@
     "html-webpack-plugin": "^3.2.0",
     "jest": "^24.8.0",
     "mini-css-extract-plugin": "^0.7.0",
+    "moxios": "^0.4.0",
+    "react-redux": "^7.0.3",
+    "react-router-dom": "^5.0.1",
```

Comment:
`react-router-dom` is included as both a dev and production dependency. You can remove it in devDependency

Reply:
Thank you @niyobobo It is solved now.